### ISSUE TITLE
model inference of version2

### ISSUE THREAD

**Issue by suzhenghang** [ExponentialML/Text-To-Video-Finetuning#50]
After the fine-tuning of version 2 is completed, how to perform model inference? version1 is as the following:

import torch
from diffusers import DiffusionPipeline, DPMSolverMultistepScheduler
from diffusers.utils import export_to_video

my_trained_model_path = "./trained_model_path/"
pipe = DiffusionPipeline.from_pretrained(my_trained_model_path, torch_dtype=torch.float16, variant="fp16")
pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
pipe.enable_model_cpu_offload()

prompt = "Your prompt based on train data"
video_frames = pipe(prompt, num_inference_steps=25).frames

out_file = "./my_video.mp4"
video_path = export_to_video(video_frames, out_file)

**Comment by xiao2mo** [comment]
mark

**Comment by ImBadAtNames2019** [comment]
> After the fine-tuning of version 2 is completed, how to perform model inference? version1 is as the following:
> 
> import torch from diffusers import DiffusionPipeline, DPMSolverMultistepScheduler from diffusers.utils import export_to_video
> 
> my_trained_model_path = "./trained_model_path/" pipe = DiffusionPipeline.from_pretrained(my_trained_model_path, torch_dtype=torch.float16, variant="fp16") pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config) pipe.enable_model_cpu_offload()
> 
> prompt = "Your prompt based on train data" video_frames = pipe(prompt, num_inference_steps=25).frames
> 
> out_file = "./my_video.mp4" video_path = export_to_video(video_frames, out_file)

Did you manage to finetune it correctly? The finetuning process makes no difference to me, it leaves the model identical to how it was before, no changes.

**Comment by suzhenghang** [comment]
I used Accelerate for distributed training, which may have differences when saving checkpoints. I need to check it. Training and validation are both fine. However, when using version 1 of the code, there's an error about initialize the models as followings:
[log.txt](https://github.com/ExponentialML/Text-To-Video-Finetuning/files/11194545/log.txt)

**Comment by suzhenghang** [comment]
@ExponentialML any suggestions ？

**Comment by ExponentialML** [comment]
> @ExponentialML any suggestions ？

Hey all, apologies for the training nuances. I've been following this thread and found that there was a bug in the way they checkpoints were being saved. The latest release should fix these issues, as well as the addition of full LoRa training.

**Comment by suzhenghang** [comment]
@ExponentialML  Nice, inference code is running normally now.
